server/src/models/Sandwich.js

```diff
@@ -24,6 +24,22 @@ class Sandwich extends unique(Model) {
     }
   }
 
+  async averageRating() {
```

Comment:
for consistency, put a `$` on this averageRating method, to help indicate it is an instance method. This is to follow the convention that Objection established